### ISSUE TITLE
Slide Not Working Right

### ISSUE THREAD

**Issue by epochslee** [Lunrtick/vue-bulma-accordion#14]
Is anyone else having issues where the slide is sliding underneat any content below the accordion? 

Also the icon is not showing in Chrome

**Comment by Hiteshpandey** [comment]
yes, I am facing the same issue. For now, i've just reduced the animation duration to 0.09s so that it is not as noticeable.

**Comment by Lunrtick** [comment]
Can you guys share an example and/or a screencap showing the issue? I haven't run into it myself.

**Comment by noahhai** [comment]
Have the same issue
System: Chrome Version 87.0.4280.141 (Official Build) (x86_64)

Config:
:caretAnimation="{
        duration: '1s',
        timerFunc: 'ease-in-out'
      }"
      :slide="{
        duration: '1s',
        timerFunc: 'ease'
      }"

![bug-bulma-accordion](https://user-images.githubusercontent.com/8117605/105631380-819de200-5e4e-11eb-8634-f6ff673a489f.gif)
